calendar.js

```diff
@@ -162,4 +171,12 @@ $(document).ready(function() {
         throw err;
       });
   }
+
+  // This is the default bucket id for the ``public:notsecret`` user on kinto.dev
+  var channelName = '94b660c6-68f3-83c4-97fa-8e8ac4715204-kinto_demo_calendar-record';
```

Comment:
Hum. I prefer this way actually : https://github.com/leplatrem/cliquet-pusher/blob/master/demo/map.js#L102-L115

If you don't (which I would understand since concise code is cool), then put it into a top variable, and use it to build the channel name. Same for collection id actually :)